### PR TITLE
remove unnecessary ACCESS_URI env var

### LINKED ISSUE #11: ACCESS_URI is unnecessary
The same information can be retrieved from the http Host request header

for example: https://github.com/matthuisman/frndlytv-for-channels/blob/276bf3c074d9fe8485cf58abfe294dc532e013c1/app.py#L65

### PR DESCRIPTION
**Pull request by tmm1**
fixes #11

### CODE REVIEW

**Comment by m0ngr31** [comment]
Thanks for the PR!

> 'http://'+req.headers.host

Can't assume it's going to be http. But I'm guessing you can pull that off of the request as well. I think I'd prefer to do string interpolation there too.

**Comment by tmm1** [comment]
Sure you can use `req.protocol`

**Comment by m0ngr31** [comment]
Something like that then?

**Comment by tmm1** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -47,5 +46,5 @@ The recommended way of running is to pull the image from [Docker Hub](https://hu
 By default, the easiest way to get running is:
 
 ```bash
-docker run -p 8000:8000 -v /config_dir:/app/config -v /dev/shm:/app/tmp -e ACCESS_URI='http://192.168.0.10:8000' -e ESPN_USER='...' -e ESPN_PASS='...'  m0ngr31/eplustv
-```
\ No newline at end of file
+docker run -p 8000:8000 -v /config_dir:/app/config -v /dev/shm:/app/tmp -e ESPN_USER='...' -e ESPN_PASS='...'  m0ngr31/eplustv
```
Comment (tmm1) [inline README.md]: single quotes apparently don't work on Windows

```suggestion
docker run -p 8000:8000 -v /config_dir:/app/config -v /dev/shm:/app/tmp -e ESPN_USER="..." -e ESPN_PASS="..."  m0ngr31/eplustv
```

Comment (tmm1) [reply README.md]: See https://community.getchannels.com/t/espn-with-custom-channels/31144/38?u=tmm1

Comment (m0ngr31) [reply README.md]: Oh, I didn't realize this had been posted there :) I love Channels!

**Pull request merged**